bms_receiver.c

```diff
@@ -0,0 +1,201 @@
+/**********************************************
+Include header files
+***********************************************/
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include "bms_receiver.h"
+
+/***********************************************************************************************************************
+Variable declaration
+**************************************************************************************************************************/
+float MinimumAttributeValueArray[NumberOfAttributes]= {100,100};
+float MaximumAttributeValueArray[NumberOfAttributes]= {0,0};
+float Temperature[Max_ArraySize]={};
+float SoC[Max_ArraySize]={};
+int NoOfEntery=0;
```

Comment:
Naming: `NoOfEntery` is the number of what entries? Incoming data up to now? total data?